motor/core.py

```diff
@@ -105,6 +105,8 @@ class AgnosticClient(AgnosticBaseProperties):
     arbiters = ReadOnlyProperty()
     close = DelegateMethod()
     __hash__ = DelegateMethod()
+    __next__ = DelegateMethod()
+    next = DelegateMethod()
```

Comment:
Do we need these methods on Client/DB/Coll? I'd prefer _not_ to add the to motor unless we absolutely need to. Eventually the plan is to remove them from pymongo in 5.0 so I don't see a reason to add them to motor.

Reply:
Sounds good, I should be able to skip the relevant tests instead